src/RoomsSDKAdapter.integration.test.js

```diff
@@ -0,0 +1,47 @@
+import {removeTestUser} from '@webex/test-users';
+
+import {createIntegrationTestUser, createIntegrationSdkInstance} from './testHelper';
+import RoomsSDKAdapter from './RoomsSDKAdapter';
+
+describe('Rooms SDK Adapter Interface', () => {
+  let roomsSDKAdapter;
+  let sdk;
+  let user;
+  let createdRoom;
+
+  beforeAll(async () => {
+    // Setting timeout higher for user creation and room creation
+    jest.setTimeout(15000);
+
+    user = await createIntegrationTestUser();
+
+    sdk = createIntegrationSdkInstance(user.token);
+
+    createdRoom = await sdk.rooms.create({title: 'Webex Teams Test Room'});
+  });
+
+  afterAll(() =>
+    removeTestUser(user).catch((reason) => {
+      // eslint-disable-next-line no-console
+      console.warn('failed to delete test user', reason);
+    })
+  );
+
+  beforeEach(() => {
+    roomsSDKAdapter = new RoomsSDKAdapter(sdk);
+  });
+
+  describe('getRoom() functionality', () => {
+    test('returns a room in a proper shape', (done) => {
+      roomsSDKAdapter.getRoom(createdRoom.id).subscribe((room) => {
+        expect(room.ID).toBe(createdRoom.id);
```

Comment:
I thought we agreed to use [`toMatchObject`](https://jestjs.io/docs/en/expect#tomatchobjectobject) instead of multiple `expect`s